lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -78,12 +87,15 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges {
     constructor(protected formCloudService: FormCloudService,
                 protected formService: FormService,
                 private notificationService: NotificationService,
+                private formRenderingService: FormRenderingService,
                 protected visibilityService: WidgetVisibilityService) {
         super();
 
         this.formService.formContentClicked.subscribe((content: any) => {
             this.formContentClicked.emit(content);
         });
+        this.formRenderingService.setComponentTypeResolver('upload', () => UploadCloudWidgetComponent, true);
```

Comment:
@SilviuCPopa @mauriziovitale Every time this component is created, it rewrites the `upload` and `dropdown` entries for the entire application. Meaning opening the Form after closing this one will still mean using cloud versions. If you are not fine with that - make a copy of the `FormRenderingService` by declaring it in the `providers` section of this component.

Reply:
@DenysVuika added FormRenderingService as provider. Thanks